File: 2016/Python/J1.py
w = 0
for i in range(6):
    g = input()
    if g == 'W':
        w+=1

if w == 5 or w == 6:
    print (1)
elif w == 3 or w == 4:
    print (2)
elif w == 1 or w == 2:
    print (3)
else:
    print(-1)

File: 2016/Java/J1.java
import java.util.*;
public class Main {
	public static void main(String args[]) {
        Scanner input = new Scanner(System.in);
        char scores [] = new char [6];
        int wCount = 0;
        for (int i = 0; i<6; i++){
            scores[i] = input.next().charAt(0);
            if (scores[i] == 'W'){
                wCount++;
            }
        }
        if (wCount == 6 || wCount == 5){
            System.out.println(1);
        }
        else if (wCount == 4 || wCount == 3){
            System.out.println(2);
        }
        else if (wCount == 2 || wCount == 1){
            System.out.println(3);
        }
        else{
            System.out.println(-1);
        }
    }
}
